### PR TITLE
Move permission request from AppDelegate to CCManageCameraUpload

### PR DESCRIPTION
**Pull request by rygen**
Using the latest API.

### CODE REVIEW

**Comment by enoch85** [comment]
@marinofaggiana ping

**Comment by jancborchardt** [comment]
@marinofaggiana please review this :)

@rygen I'll add you to the Nextcloud iOS team so you can directly open branches in the main repo :)

**Comment by marinofaggiana** [comment]
please wait ..  several modify in progress :-)

**Comment by rygen** [comment]
@jancborchardt  thanks :)
@marinofaggiana it might be conflicting due to now lagging several commits behind master. I can redo it if it facilitates the process :)

**Comment by enoch85** [comment]
@rygen If there would be conflicts it would have shown in this PR afaik.

File: iOSClient/Settings/CCManageCameraUpload.m [hunk 1]
```diff
@@ -62,6 +62,20 @@ - (void)initializeForm
 
     form = [XLFormDescriptor formDescriptorWithTitle:NSLocalizedString(@"_uploading_from_camera_", nil)];
     
+    // Request permission for camera roll access
```
Comment (timbru31) [inline iOSClient/Settings/CCManageCameraUpload.m]: IMHO comment not needed

Comment (rygen) [inline iOSClient/Settings/CCManageCameraUpload.m]: Agreed. But I do think this pull request contributes to a relevant UX improvement. :)

Comment (marinofaggiana) [inline iOSClient/Settings/CCManageCameraUpload.m]: sorry boy, today I path this :)

Comment (rygen) [inline iOSClient/Settings/CCManageCameraUpload.m]: 🎉 :))

**Comment by marinofaggiana** [comment]
OK  Merge manually for conflicts   

Thanks @rygen

**Comment by jancborchardt** [comment]
Btw @rygen I just saw you are based in Berlin – you are very welcome at our regular Berlin meetup: https://www.meetup.com/nextcloud-berlin and at our upcoming conference: https://www.meetup.com/nextcloud-berlin/events/238587046/

**Comment by rygen** [comment]
@jancborchardt I'll definitely try to attend the next series of the meetup, likely already the next one. Registered for the conference when I received the email :)

**Comment by jancborchardt** [comment]
@rygen cool! Welcome to the community! :)

**Pull request closed without merging**